src/lib/__tests__/InstantSearch-test.js

```diff
@@ -995,10 +999,23 @@ describe('createURL', () => {
     createURL: jest.fn(() => '#'),
   });
 
-  it('returns the URL for the main index state', () => {
+  it('at top-level returns the default URL for the main index state', () => {
     const router = createRouter();
-    router.createURL.mockImplementation(() => 'http://algolia.com');
```

Comment:
I would have kept this, at least we know where the value come from.

Reply:
82c05556460545f474bc507267f7218198ba284c